cpp/tests/utilities/timestamp_utilities.cuh

```diff
@@ -70,8 +70,9 @@ generate_timestamps(int32_t count, time_point_ms start, time_point_ms stop) {
     auto mask = cudf::test::make_counting_transform_iterator(0, [](auto i) { return i % 2 == 0; });
     return cudf::test::fixed_width_column_wrapper<T>(iter, iter + count, mask);
   }
-
-  return cudf::test::fixed_width_column_wrapper<T>(iter, iter + count);
+  else {
```

Comment:
This else is to quash a "statement unreachable" warning.

Reply:
Maybe add a comment here so someone does not try to remove it later.